tutor/src/screens/assignment-edit/ux.js

```diff
@@ -31,6 +31,9 @@ export default class AssignmentUX {
   @observable templates;
   @observable plan;
 
+  didUserChangeDatesManually = false;
+  dueAt;
```

Comment:
I think these should probably be `@observable` so that any `@computed` values that depend on them know to re-calculate